sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/DiagnosticsClientContext.java

```diff
@@ -188,7 +188,7 @@ private String gwConfigInternal() {
             }
             return Strings.lenientFormat("(cps:%s, rto:%s, icto:%s, p:%s)",
                 this.httpClientConfig.getMaxPoolSize(),
-                this.httpClientConfig.getRequestTimeout(),
+                this.httpClientConfig.getNetworkRequestTimeout(),
```

Comment:
how about changing the diagnostics name from rto to nrto (NetworkRequestTimeout

(cps:%s, rto:%s, icto:%s, p:%s) -> (cps:%s, nrto:%s, icto:%s, p:%s)

Reply:
User should not be able to configure the gateway timeout. The gateway may need to do retries because of 410s or other scenarios which is why the .NET SDK has a hard coded value of 60 seconds.

Reply:
Users can customize but never under 60 seconds.

Reply:
I think that's what Java V2 had when exposed? @simplynaveen20 might remember better

Reply:
Currently in v4, there is no public Api for requestTimeout. Same plan for networkRequestTimeout.

And the real http requestTimeout by default is 60s, which can be override by system property.

Reply:
updated rto -> nrto.